interpreter/core/core.py

```diff
@@ -30,6 +30,7 @@ def __init__(self):
         self.auto_run = False
         self.debug_mode = False
         self.max_output = 2000
+        self.scan_code = "off"
```

Comment:
I wasn't sure if we'd want to put this under some sort of larger `security`/`safety` subsection so that the `config.yaml` is a little easier to understand and doesn't become too polluted with top-level keys as more features (like using `guarddog` to scan dependencies before installing them) are added.

Maybe something like:

```yaml
system_message: |
  You are Open Interpreter, a world-class programmer that can complete any goal by executing code.
  ...
local: false
model: "gpt-4"
temperature: 0
security:
  scan_code: "ask"
```

But I didn't want to start fiddling with the config parsing logic if I didn't need to.